docs/examples/Station.ipynb

```diff
@@ -527,11 +528,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qcodes.utils.installation import register_station_schema_with_vscode\n",
+    "# from qcodes.utils.installation import register_station_schema_with_vscode\n",
```

Comment:
commented out?

Reply:
Noooo...  ;-( That was a leftover from playing with different options to inline a yaml file.